### PR TITLE
Fix 'Refactor CI/CD files' action failure

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
# Fix 'Refactor CI/CD files' action failure

**_WHAT DOES THIS PR DO?_**
Fixes 'Refactor CI/CD files' action failure

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
- n/a

**_WHAT IS THE LINK TO THE PR BRANCH_**
[fix-refactorCICD-failure](https://github.com/airqo-platform/AirQo-api/tree/fix-refactorCICD-failure)

**_HOW DO I TEST OUT THIS PR?_**
-n/a

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
-n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
-n/a

**_ARE THERE ANY RELATED PRs?_**
[Refactor CI/CD files](https://github.com/airqo-platform/AirQo-api/pull/712)

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike cc: @NoahNsimbe

File: .github/workflows/deploy-apis-to-staging.yml [hunk 1]
```diff
@@ -509,7 +509,7 @@ jobs:
       - name: Deploy to K8S
         run: |
           cd k8s/device-status/
-          kubectl apply -f stage-device-status-job.yaml
+          kubectl apply -f stage-airqo-device-status-job.yaml
           kubectl rollout restart deployment/airqo-stage-device-status-job -n staging
```
[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply .github/workflows/deploy-apis-to-staging.yml:513]: Thanks @NoahNsimbe , let me look into it

[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply .github/workflows/deploy-apis-to-staging.yml:513]: @NoahNsimbe

[the remark this answers is not in the document]
Comment (NoahNsimbe) [reply .github/workflows/deploy-apis-to-staging.yml:513]: This is what I see from my end. Wondering whether that wont affect the deployment
Screenshot 2022-02-25 at 16 10 42

[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply .github/workflows/deploy-apis-to-staging.yml:513]: Thanks @NoahNsimbe, let me examine this shortly.

[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply .github/workflows/deploy-apis-to-staging.yml]: Thanks again @NoahNsimbe 
I've made the correction

**Pull request merged**